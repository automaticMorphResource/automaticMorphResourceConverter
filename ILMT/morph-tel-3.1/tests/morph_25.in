1	kAnI	unk	
1	ippudu	unk
1	avi	unk	
1	kanipiMcadaM	unk
1	lexu	unk	
1	.	unk	
