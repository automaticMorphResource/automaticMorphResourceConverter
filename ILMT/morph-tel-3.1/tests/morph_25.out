1	kAnI	unk	<fs af='kAnI,avy,,,,,0,0_avy'>|<fs af='kAni,avy,,,,,V,V_avy'>|<fs af='kAni,avy,,,,,V,V_avy'>
2	ippudu	unk	<fs af='ippudu,n,,sg,,d,0,0'>|<fs af='ippudu,n,,sg,,o,ti,ti'>
3	avi	unk	<fs af='axi,pn,fn,pl,3,d,0,0'>
4	kanipiMcadaM	unk	<fs af='kanipiMcu,v,any,any,any,,adaM,adaM'>
5	lexu	unk	<fs af='gala,v,fn,sg,3,,a,a'>|<fs af='uMdu,v,any,any,any,,a_lexu,a_lexu'>
6	.	unk	<fs af='&dot;,punc,,,,,,'>
